vault/ha.go

```diff
@@ -6,6 +6,8 @@ import (
 	"crypto/x509"
 	"errors"
 	"fmt"
+	"github.com/aws/aws-sdk-go/aws/awserr"
```

Comment:
What's the reason for these imports?

Reply:
Ah, that should have been removed when I moved the DynamoDB specific logic back into dyanamodb.go.